x-pack/plugins/enterprise_search/public/applications/app_search/components/app_search_gate/app_search_gate_logic.ts

```diff
@@ -52,6 +55,9 @@ export const AppSearchGateLogic = kea<MakeLogicType<AppSearchGateValues, AppSear
         });
       }
     },
+    submitGatedFormSuccess: () => {
+      window.location.reload();
```

Comment:
Not sure if this is the best option, but it worked.

Reply:
Thanks for this @TattdCodeMonkey - I'll be sure to thoroughly test this tomorrow (Monday - unless someone else gets to it first).

As far as I understand from the research I did on this, it's really the `e.preventDefault();` that helps too (could certainly be different in this case!) as it seems to be the reload that causes the network connection to break and fail...